generated/nimodinst/setup.py

```diff
@@ -45,6 +45,7 @@ def read_contents(file_to_read):
     install_requires=[
         'enum34;python_version<"3.4"',
         'singledispatch;python_version<"3.4"',
+        'hightime',
```

Comment:
This is a bit of a shame - nimodinst truly doesn't use any timedeltas yet we're installing it.

Reply:
Typically user applications will include other nimi-python modules along with nimodinst. In the end, maybe it doesn't really matter that we have an unnecessary dependency?